core/services/vrf/listener_v2.go

```diff
@@ -585,12 +596,18 @@ func (lsn *listenerV2) processRequestsPerSubBatch(
 				"attempts", p.req.attempts)
 
 			if p.err != nil {
-				if startBalanceNoReserveLink.Cmp(p.juelsNeeded) < 0 {
-					ll.Infow("Insufficient link balance to fulfill a request based on estimate, returning")
+				if startBalanceNoReserveLink.Cmp(p.juelsNeeded) < 0 && errors.Is(p.err, errPossiblyInsufficientFunds{}) {
+					ll.Infow("Insufficient link balance to fulfill a request based on estimate, returning", "err", p.err)
 					return processed
 				}
 
-				ll.Errorw("Pipeline error", "err", p.err)
+				if errors.Is(p.err, errBlockhashNotInStore{}) {
+					// Running the blockhash store feeder in backwards mode will be required to
+					// resolve this.
+					ll.Criticalw("Pipeline error", "err", p.err)
```

Comment:
Would it be worth metioning some part of the comment in the log message? Or will the node op already know what to do?
(Same Q for line 736)

Reply:
I think the relevant error msg will show up in the error fields `err="Blockhash not found in store"` which should be enough for on-call engineers to come to the correct conclusion.

Reply:
But I guess we can make it more visible and put it in the main log message.